### PR TITLE
SPORTS-50 Make basic layout for fixtures and results

### PR DESCRIPTION
**Pull request by Ishanliv99**

### CODE REVIEW

File: frontend/src/assets/sass/modules/_fixture.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    width: 720px;
+    padding-bottom: 120px;
+  }
+
+  .player-mirror {
+    .player-image-container {
+      right: -40px;
+    }
+  }
+}
```
Comment (aviskarkc10) [inline frontend/src/assets/sass/modules/_fixture.scss]: new line at eof

File: frontend/src/components/commons/header/Header.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import logo from '../../../../public/assets/images/sports-logo.png';
+
+const Header = (props) => {
+  return(
+    <div className="header-main-container">
+      <div className="header-content-container">
+        <div className="logo-container">
+          <a href = "#">
+          <img src={logo} alt="Sports logo"/>
```
Comment (aviskarkc10) [inline frontend/src/components/commons/header/Header.js]: fix indentation

File: frontend/src/components/user/User.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Header from '../commons/header';
+import Title from '../commons/title';
+import Dates from '../commons/date';
+import Fixture from '../fixture';
```
Comment (aviskarkc10) [inline frontend/src/components/user/User.js]: length rule

File: frontend/src/assets/sass/modules/_date.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.dates-container {
+    padding: 20px 0px;
+    text-align: center;
+    font-family: "Montserrat";
+}
+
+.dates {
+    font-size: 36px;
+    color: #2ECC71;
```
Comment (aviskarkc10) [inline frontend/src/assets/sass/modules/_date.scss]: length rule

File: frontend/src/assets/sass/utilities/_config.scss [hunk 5]
```diff
@@ -200,3 +200,14 @@ $playoff-container-height: 526px;
...
+
+//Logo
+$logo-text-separation-border-style: solid;
+$logo-text-separation-border-color: #E5E5E5;
+
+//Header
+$header-width: 190px;
+$header-padding: 20px;
+$header-font-size: 22px;
+$header-color: #737373;
```
Comment (aviskarkc10) [inline frontend/src/assets/sass/utilities/_config.scss]: length rule

File: frontend/src/assets/sass/modules/_fixture.scss [hunk 6]
```diff
@@ -0,0 +1,28 @@
+.fixture-container {
+    width: 508px;
+    padding: 40px 0px;
+    position: relative;
+}
+
+.player-mirror {
+  @extend %content-right;
+  .player-image-container{
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_fixture.scss]: Add new line here.

File: frontend/src/components/commons/date/Dates.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Dates = (prop) => {
```
Comment (aviskarkc10) [inline frontend/src/components/commons/date/Dates.js]: let's use var name props

File: frontend/src/assets/sass/modules/_fixture.scss [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    right: 0px;
+  }
+  .player-name-container{
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_fixture.scss]: Add new line here.

File: frontend/src/components/fixture/Fixture.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Player from '../player';
+import Time from '../time';
```
Comment (aviskarkc10) [inline frontend/src/components/fixture/Fixture.js]: length rule

File: frontend/src/assets/sass/modules/_header.scss [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    @extend %header-container;
+}
+
+.header-content-container {
+    @extend %header-container-content;
+}
+
+.logo-container {
+    @extend %logo;
+    float: left;
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_header.scss]: new line here.

File: frontend/src/assets/sass/modules/_player.scss [hunk 11]
```diff
@@ -0,0 +1,55 @@
+.player-main-container{
+  width: 244px;
+  position: relative;
+}
+
+.player-image-container {
+  z-index: 1;
+  position: absolute;
+  img {
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_player.scss]: new line here

File: frontend/src/assets/sass/modules/_user.scss [hunk 12]
```diff
@@ -0,0 +1,38 @@
+.user-main-container {
+  padding: 30px;
+}
+
+.user-left-container {
+  @extend %content-left;
+  width: 550px;
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_user.scss]: new line here

File: frontend/src/assets/sass/modules/_user.scss [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.user-top-left-container {
+  margin: 40px 0px;
+}
+
+.user-right-container {
+  @extend %content-right;
+  width: 550px;
```
Comment (arun-tamang) [inline frontend/src/assets/sass/modules/_user.scss]: new line here.

File: frontend/src/assets/sass/utilities/_config.scss [hunk 14]
```diff
@@ -200,3 +200,14 @@ $playoff-container-height: 526px;
 $semi-final-container-height: 306px;
 $quarter-final-container-height: 322px;
 $round-of-sixteen-container-height: 540px;
+
+//Logo
```
Comment (arun-tamang) [inline frontend/src/assets/sass/utilities/_config.scss]: Add space after double slash

File: frontend/src/assets/sass/utilities/_config.scss [hunk 15]
```diff
@@ -200,3 +200,14 @@ $playoff-container-height: 526px;
...
+$logo-text-separation-border-style: solid;
+$logo-text-separation-border-color: #E5E5E5;
+
+//Header
```
Comment (arun-tamang) [inline frontend/src/assets/sass/utilities/_config.scss]: space required in comment

File: frontend/src/assets/sass/utilities/_helpers.scss [hunk 16]
```diff
@@ -6,10 +6,7 @@
 // will have an !important property attached to it, so that a helper class cannot be overwritten.
 // Placeholders can also be included in the helper partial for these purposes.
 // ==========================================================================
-
-
 //vertical centering helpers
```
Comment (arun-tamang) [inline frontend/src/assets/sass/utilities/_helpers.scss]: Start comment with capital letter. and add space before starting.

File: frontend/src/assets/sass/utilities/_helpers.scss [hunk 17]
```diff
@@ -26,8 +23,9 @@
 }
 
 %inline-after {
-  &::after {// override global clearfix
-      display: inline;
+  &::after {
+    // override global clearfix
```
Comment (arun-tamang) [inline frontend/src/assets/sass/utilities/_helpers.scss]: Start comment with capital letter.

File: frontend/src/assets/sass/utilities/_helpers.scss [hunk 18]
```diff
@@ -59,3 +57,57 @@
...
+
+%content-left {
+  float: left;
+}
+
+%content-right {
+  float: right;
+}
+
+// login page helpers
```
Comment (arun-tamang) [inline frontend/src/assets/sass/utilities/_helpers.scss]: Capitalize first letter.

File: frontend/src/components/fixture/Fixture.js (hunk 9 above)
Comment (arun-tamang) [inline frontend/src/components/fixture/Fixture.js]: ladder

File: frontend/src/components/fixture/Fixture.js [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+const Fixture = (props) => {
+  return(
+    <div className="fixture-container">
+      <Player/>
+      <Time/>
+      <span className="player-mirror"><Player/></span>
+    </div>
+  );
+};
+export default Fixture;
```
Comment (arun-tamang) [inline frontend/src/components/fixture/Fixture.js]: new line here.

File: frontend/src/components/player/Player.js [hunk 20]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import image from '../../../public/assets/images/image.jpg';
+
+
```
Comment (arun-tamang) [inline frontend/src/components/player/Player.js]: delete extra line

**Pull request merged**